src/Sparrow/Json/ManualBlittableJsonDocumentBuilder.cs

```diff
@@ -572,7 +587,8 @@ public unsafe void WriteRawBlob(byte* ptr, int size)
         public unsafe void WriteEmbeddedBlittableDocument(byte* ptr, int size)
         {
             var currentState = _continuationState.Pop();
-            var valuePos = _writer.WriteValue(ptr, size, out _, UsageMode.None, null);
+            // var valuePos = _writer.WriteValue(ptr, size, out _, UsageMode.None, null);
```

Comment:
why is this commented?

Reply:
Because I was working on it, and forgot to delete it.